pgx/src/datum/internal.rs

```diff
@@ -43,10 +50,28 @@ impl Internal {
     /// your responsibility.
     #[inline]
     pub unsafe fn get<T>(&self) -> Option<&T> {
-        self.0.map(|datum| (datum as *const T).as_ref()).flatten()
+        self.0.and_then(|datum| (datum as *const T).as_ref())
```

Comment:
Does this change our minim rust version?  Not that I care but if it does we need to document it

Reply:
I don't think so, these have been around awhile.